R/ui.R

```diff
@@ -0,0 +1,132 @@
+#' @include globals.R
+
+# Mount OpenAPI and UI
+#' @noRd
+mountUI <- function(pr, host, port, ui, callback, ...) {
```

Comment:
Pass in args as a list

```suggestion
mountUI <- function(pr, host, port, ui, callback, args = list()) {
```